bugbear.py

```diff
@@ -954,6 +955,36 @@ def check_for_b031(self, loop_node):  # noqa: C901
                                 B031(node.lineno, node.col_offset, vars=(node.id,))
                             )
 
+    def _get_names_from_tuple(self, node: ast.Tuple):
+        for dim in node.dims:
```

Comment:
Is this >= 3.9 only?

Reply:
yup, fixed. Sorry, didn't have actions enabled on my fork, got that working now too